RxRestClient/Classes/Models/BaseResponse.swift

```diff
@@ -19,13 +19,13 @@ import Foundation
 public enum BaseResponse {
     case serviceOffline
 
-    case badRequest
+    case badRequest(body: String?)
```

Comment:
We can have non-optional Strings for bodies as we are getting non-optional from a response.